modules/rosa-creating-node-tuning.adoc

```diff
@@ -0,0 +1,106 @@
+// Module included in the following assemblies:
+//
+// * rosa_hcp/rosa-tuning-config.adoc
+
+:_content-type: PROCEDURE
+[id="rosa-creating-node-tuning_{context}"]
+= Creating node tuning configurations on {hcp-title}
+
+You can create tuning configurations using the `rosa` CLI.
+
+.Prerequisites
+
+* You have downloaded the latest version of the ROSA CLI.
+* You have a cluster on the latest version 
+
+.Procedure
+
+. To create the tuning configurations, you must have an existing specification file. The following example shows a usable specification which is saved as `spec.json`:
++
+[source,json]
+----
+{
+  "profile": [
+    {
+      "data": "[main]\nsummary=Custom OpenShift profile\ninclude=openshift-node\n\n[sysctl]\nvm.dirty_ratio=\"55\"\n", <1>
+      "name": "tuned-1-profile" <2>
+    }
+  ],
+  "recommend": [
+    {
+      "priority": 20,
+      "profile": "tuned-1-profile"
+    }
+  ]
+}
+----
++
+The following items in this spec file are:
++
+<1> Item 1
+<2> Item 2
+
+. After creating your specification file, run the following command:
++
+[source,terminal]
+----
+$ rosa create tuning-config -c <cluster_id> --name <name_of_tuning> --spec-path <path_to_spec_file>
+----
++
+You must supply the path to the `spec.json` file or the command returns an error.
++
+.Sample output
+[source,terminal]
+----
+$ I: Tuning config 'sample-tuning' has been created on cluster 'cluster-example'.
+$ I: To view all tuning configs, run 'rosa list tuning-configs -c cluster-example'
+----
+
+.Validation
+
+* You can verify the existing tuning configurations that are applied by your account with the following command:
++
+[source,terminal]
+----
+$ rosa list tuning-configs -c <cluster_name> [-o json]
+----
++
+You can specify the type of output you want for the configuration list.
+
+** Without specifying the output type, you see the ID and name of the tuning configuration:
++
+.Sample output without specifying output type
+[source,terminal]
+----
+ID                                    NAME
+20468b8e-edc7-11ed-b0e4-0a580a800298  sample-tuning
+----
+
+** If you specify an output type, such as `json`, you receive the tuning configuration as JSON text:
```

Comment:
Note that there is also a 
```
rosa describe tuning-config -c <cluster-name> <tuning-config-name>
```
which shows the same thing without having a full JSON output

Reply:
ok you mentioned it later, but could be worth having it here too